src/org/traccar/database/DeviceManager.java

```diff
@@ -52,6 +55,13 @@
 
     private final Map<Long, Position> positions = new ConcurrentHashMap<>();
 
+    private final Map<Long, Boolean> motionStates = new ConcurrentHashMap<>();
```

Comment:
It is faster than queries each parameter separate and it will enable us to easily add some extra information later.